pkg/manager/getconfig.go

```diff
@@ -47,6 +50,33 @@ func (m *Manager) GetTargetConfig(target string, configname string, path string,
 				fmt.Errorf("No Configuration found for %s", configname)
 		}
 	}
+
+	readOnlyPaths, ok := m.ModelRegistry.ModelReadOnlyPaths[utils.ToModelName(config.Type, config.Version)]
+
+	if !ok {
+		log.Warningf("Cannot check for Read Only paths for %s %s because "+
+			"Model Plugin not available - continuing", config.Type, config.Version)
+	} else {
+		//TODO does this account for list values?
+		log.Info("path ", path)
+		log.Info("readonlypaths ", readOnlyPaths)
+		log.Info("Op state cache", m.OperationalStateCache[topocache.ID(target)])
+		_, ok := readOnlyPaths[path]
```

Comment:
is this checking the path is valid or something?

Reply:
Yes, but that has to be incremented to include a LPM for the path, it won't match as is.